source/game.screen.programmeplanner.bmx

```diff
@@ -690,18 +697,22 @@ Type TScreenHandler_ProgrammePlanner
 
 		Local item:TGUIProgrammePlanElement = TGUIProgrammePlanElement(triggerEvent.GetSender())
 		If Not item Then Return False
-
+		DebugPrint("onTryDragProgrammePlanElement -> item="+item.ToString())
```

Comment:
Sollen die print-Aufrufe drinbleiben? Auch wenn sie zentral deaktiviert sind, finden die Aufrufe statt. An anderen Stellen ist sowas auskommentiert oder nur in debug-Builds aktiviert.

Reply:
Naja ich will sie nicht als "DebugLog" drin haben (dann waeren sie nur in DebugBuilds drin). Aber wenn ich sie "auskommentiere", dann brauche ich ja den zentralen Schalter nicht.

Da es sich hier um einen Aufruf handelt, der nur bei einem ausgeloesten Event auftritt, sollte der Impact gering ausfallen.

Kann das aber gerne auch alles einfach entfernen.

Reply:
Es scheint sich ja wirklich nur um "Helfer" für die Entwicklung zu handeln. Wenn's funktioniert, braucht man die Ausgaben ja auch nicht reaktivierbar. Ich würde sie rauswerfen.